graviton/inspector.py

```diff
@@ -504,7 +504,7 @@ def budget_consumed(self) -> Optional[int]:
         """
         return self.dig(DRProp.budgetConsumed) if self.is_app() else None
 
-    def last_log(self) -> Optional[str]:
+    def last_log(self) -> Any:
```

Comment:
the change in type is because `dig` is returning `Any`, right?

Reply:
Yes!

Reply:
okok, I think I am fine with the move, for no behavior change behind the curtain.